meta-mender-core/recipes-mender/mender-client/mender-client.inc

```diff
@@ -50,62 +41,57 @@ FILES:${PN} += "\
     ${datadir}/mender/modules/v3/script \
     ${datadir}/mender/modules/v3/single-file \
     ${sysconfdir}/udev/mount.blacklist.d/mender \
-    ${systemd_unitdir}/system/${MENDER_CLIENT}.service \
     /data/mender/device_type \
     /data/mender/mender.conf \
 "
+
+FILES:${PN} += "\
```

Comment:
Is `mender-client.inc` shared between Go and C++ variants? I suspect that this will lead to problems because `${PN}` will always be different for the two.